src/components/reports/presentation-video-report.js

```
@@ -12,7 +12,7 @@
  **/
 
 import React from "react";
-import { Table } from "openstack-uicore-foundation/lib/components";
+import Table from "openstack-uicore-foundation/lib/components/table";
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)